app/controllers/sitters_controller.rb

```diff
@@ -7,7 +7,15 @@ def index
       redirect_to root_path
     else
       @location = coordinates_for(params[:location])
-      @sitters = User.where(sitter: true).near(@location, 50)
+      @sitters = User.where(
+        sitter: true
+        ).near(
+        @location, 50
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Indent the first parameter one step more than the previous line.